x/incentives/keeper/gauge_test.go

```diff
@@ -146,6 +151,7 @@ func (suite *KeeperTestSuite) TestNonPerpetualGaugeOperations() {
 	suite.Require().Equal(sdk.Coins{}, rewardsEst)
 }
 
+// TestPerpetualGaugeOperations tests perpetual gauge distribution logic.
```

Comment:
This test is being completely refactored to be table-driven in this PR: https://github.com/osmosis-labs/osmosis/pull/1932